ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -334,6 +334,27 @@ def pytest_addoption(parser):
             set for installing lvmo operator and lvmo cluster
             """,
     )
+    parser.addoption(
+        "--disable-environment-checker",
+        dest="disable_environment_checker",
+        action="store_true",
+        default=False,
+        help="Disable environment checks for test cases.",
+    )
+    parser.addoption(
+        "--resource-checker",
+        dest="resource_checker",
+        action="store_true",
+        default=False,
+        help=(
+            "Resource checker checks for the left over resources which are added while running the test cases."
+        ),
+    )
+    parser.addoption(
+        "--kubeconfig",
+        dest="kubeconfig",
+        help=("Kubeconfig location which will be loaded as environmental variable"),
```

Comment:
1. We should check on exporting evn variable to overwrite the default one.
2. check the multicluster scenario and context switching how it's handled per cluster.
3. Overwrite config value under config.RUN["kubeconfig"]
4. We also have kubeconfig_location under RUN section which looks like is a path from cluster dir - just check it's usage if it's OK and will not break anything.

Reply:
> 1. We should check on exporting evn variable to overwrite the default one.
> 
>     2. check the multicluster scenario and context switching how it's handled per cluster.
> 
>     3. Overwrite config value under config.RUN["kubeconfig"]
> 
>     4. We also have kubeconfig_location under RUN section which looks like is a path from cluster dir - just check it's usage if it's OK and will not break anything.

It won't break anything. Acceptance suite is clean and this PR didn't break any existing flow.

done apart from multicluster scenario. will take multicluster scenario later.